catalog/async/catalog-async-data/src/main/java/org/codice/ddf/catalog/async/data/impl/ProcessResourceImpl.java

```diff
@@ -184,7 +196,7 @@ public boolean isModified() {
 
   @Override
   public void close() {
-    IOUtils.closeQuietly(inputStream);
+    IOUtils.closeQuietly(inputStreamDataCache);
```

Comment:
❗️ Same concurrency concerns and need to close all returned output streams as stated above.